app/models/repo.rb

```diff
@@ -0,0 +1,9 @@
+class Repo
+  attr_reader :name,
+              :html_url
+
+  def initialize(repo_data)
+    @name = repo_data[:name]
+    @html_url = repo_data[:html_url]
+  end
```

Comment:
We can add more repo_data when needed, created at etc...

Reply:
It looks like we definitely will. This is great :)